### ISSUE TITLE
Few feature ideas

### ISSUE THREAD

**Issue by azsenca** [whmcs-smarters/IPTV-Smarters-TV-Demo-APK#78]
1. Media flags of codec and quality. For example, show flags on playing channel, something like SD, HD, or FHD, and codec info h254, h265, aac, mp3, mp2, DD, DTS...

Example photo:
![img3](https://user-images.githubusercontent.com/1789117/41002630-8bfdf622-6914-11e8-8adb-be36e351b69a.png)

2. Do not auto-play first channel in list. It is annoying for most users.
3. When playing channel full screen, use UP and DOWN buttons on remote to change channels.
4. When playing channel full screen use long pres or menu button to show channel list on-screen

**Comment by whmcs-smarters** [comment]
Thanks for your valuable feedback. We will work on it for sure. 

When playing channel full screen, use UP and DOWN buttons on remote to change channels. = **That should be working as you mentioned**

Anyways, will cross-check and update there. 

Thanks

**Comment by azsenca** [comment]
Yeah, channel switch with UP and DOWN works now. My remote had wrong set-up.

Another good feature would be when playing channel full screen and you hold back button, long press, to switch to previous channel.
